doc-content/enterprise-only/bpmn/bpmn-gateways-ref.adoc

```diff
@@ -1,104 +1,77 @@
-= Gateways 
-Gateways are used to create or synchronize branches in the workflow using a set of conditions, which is called the gating mechanism. Gateways are of two types:
+[id='bpmn-gateways_{context}']
+= BPMN2 gateways in process designer  
+Gateways are used to create or synchronize branches in the workflow using a set of conditions called the gating mechanism. There are two types of gateways:
 
-* Converging, that is merging multiple flows into one flow.
-* Diverging, that is splitting one Flow into multiple flows.
+* Converging, merging multiple flows into one flow
+* Diverging, splitting one flow into multiple flows
 
-One Gateway cannot have multiple incoming _and_ multiple outgoing flows.
+One gateway cannot have multiple incoming and multiple outgoing flows.
+
+In the following business process diagram, the XOR gateway evaluates only the incoming flow whose condition evaluates to true:
+image:BPMN2/gateway.png[]
+
+In this example, the customer details are verified by a user and the process is assigned to a user for approval. If approved, an approval notification is sent to the user. If the event of the request is rejected, a rejection notification is sent to the user.
 
 
 .Gateway elements table
-[cols="4"]
+[cols="2"]
 |===
-.2+h|
-.2+h|Top-level
-2+h|Subprocesses
 
 
-h|Interrupt
-h|Non-interrupt 
+|exclusive (XOR)
+|image:BPMN2/bpmn-gateway-exclusive.png[]
 
-|Exclusive (XOR)
-|image:BPMN2/bpmn-start-node.png[]
-|
-|
 
 |Inclusive
-|
-|
-|
+|image:BPMN2/bpmn-gateway-inclusive.png[]
+
+
 
 |Parallel
-|
-|
-|
+|image:BPMN2/bpmn-gateway-parallel.png[]
+
 
 |Event
-|
-|
-|
+|image:BPMN2/grn_star_lg.png[]
+
 
 |===
 
 [[_exclusive_gateway]]
 .Exclusive
-In an exclusive converging gateway, only the first incoming flow whose condition evaluates to true is chosen. In a diverging gateway only one outgoing Flow is chosen.
+In an exclusive diverging gateway, only the first incoming flow whose condition evaluates to true is chosen. In a converging gateway, the next node is triggered for each triggered incoming flow.
 
 The gateway triggers exactly one outgoing flow. The flow with the constraint evaluated to true and the _lowest_ priority number is taken.
 
 
 [IMPORTANT]
 ====
-Ensure that at least one of the outgoing flow evaluates to true at runtime. Otherwise, the process instance terminates with a runtime exception.
+Ensure that at least one of the outgoing flows evaluates to true at runtime. Otherwise, the process instance terminates with a runtime exception.
```

Comment:
"true at run time" (two words when noun), though "runtime exception "is correct. Per IBM. Search all "runtime" instances to confirm.